src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/performance.html.twig

```diff
@@ -104,7 +104,16 @@
               </label>
 
               <div class="col-sm input-container">
-                <a class="btn btn-primary pointer" href="{{ path('admin_performance_module_disable_non_builtin') }}" title="Clear cache">
+                <a
+                  class="btn btn-primary pointer"
+                  href="{{ path('admin_performance_module_disable_non_builtin') }}"
+                  title="{{ 'Disable non built-in modules'|trans({}, 'Admin.Advparameters.Feature') }}"
+                  id="disableNonBuiltInModulesBtn"
+                  data-confirm-title="{{ 'It will disable all the non built-in module.'|trans({}, 'Admin.Advparameters.Help') }}"
```

Comment:
```suggestion
                  data-confirm-title="{{ 'It will disable all the non-built-in modules.'|trans({}, 'Admin.Advparameters.Help') }}"
```

a little typo fix there and it's ready 👍 + Grammarly is showing me to put `non-built-in` connected